DigitalLearningSolutions.Web/Views/Shared/_Layout.cshtml

```diff
@@ -44,8 +45,8 @@
       <div class="nhsuk-width-container nhsuk-header__container">
         <div class="nhsuk-header__logo">
           <a class="nhsuk-header__link nhsuk-header__link--service"
-             href="@(ViewData["HeaderPath"] ?? @Configuration["CurrentSystemBaseUrl"] + "/Home?action=appselect")"
-             aria-label="NHS Digital Learning Solutions - @(ViewData["HeaderPathName"] ?? "Switch Application")">
+             href="@(ViewData["HeaderPath"] ?? @Configuration["CurrentSystemBaseUrl"] + "/Home")"

```

Comment:
I think you've forgotten to change the base url here :)

Reply:
Nice spot 😄